projects/driver_controls/inc/can_output.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+// Module for broadcasting CAN messages
+#include <stdint.h>
+#include "status.h"
```

Comment:
Please run `make format` over these files.

Reply:
Sorry about the late reply. Fixed